tools/requirements.txt

```diff
@@ -2,8 +2,6 @@ ansible-core
 ncclient
 paramiko
 pyOpenSSL<20.0.0  # Transient dependency for credssp support.
-pypsrp[kerberos,credssp]
-pywinrm[kerberos,credssp]
```

Comment:
Although this fixes the immediate problem with integration tests, some EEs based on our image might be depending on having these packages available. Therefore, we can't remove them.

Speaking with nitz on this, he's identified a line in our `Dockerfile` that seems to be causing the issue. If you comment out the `ENV PIP_OPTS=--no-build-isolation` line in `Dockerfile`, the tests should succeed. The tests will take longer now, but at least we won't break anyone.